pkgs/applications/office/libreoffice/default.nix

```diff
@@ -241,9 +243,10 @@ in stdenv.mkDerivation rec {
       libxshmfence libatomic_ops graphite2 harfbuzz
       librevenge libe-book libmwaw glm glew
       libodfgen CoinMP librdf_rasqal defaultIconTheme makeWrapper
-    ];
+    ]
+    ++ lib.optional kdeIntegration KDE4DIR;
```

Comment:
A trick to accelerate your build: 

use a `langs = [ "en-US" ]` argument override to avoid building all other languages.